src/index.html

```diff
@@ -16,7 +16,92 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo-link"
+        target="_blank"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo icon"
+          width="40px"
+          height="40px"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav-list">
+          <li>
+            <a
+              class="is-active"
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              class="nav__link is-active"
```